lib/lunchbot_web/live/component/order_component.ex

```diff
@@ -0,0 +1,33 @@
+defmodule LunchbotWeb.OrderComponent do
+  @moduledoc false
+
+  use Phoenix.Component
+
+  alias Lunchbot.LunchbotData.Order
+  alias Lunchbot.LunchbotData.OrderItem
+
+  # Visual representation of an "Order".
+  # Required field:
+  #   order = Lunchbot.LunchbotData.Order
+  def display_order(assigns) do
+    ~H"""
+    <div>
+      From <b><%= assigns.order.menu.name %></b> @ <em><%= assigns.order.inserted_at %></em>
+      <ul>
+        <%= for order_item <- assigns.order.order_items do %>
+            <li><%= order_item.item.name %>
+              <%= if !Enum.empty?(order_item.order_item_options) do %>
+                (
+                <%= for order_item_option <- order_item.order_item_options do %>
+                  <%= order_item_option.option.name %>
+                <% end %>
+                )
+              <% end %>
+              for <em>$<%= OrderItem.get_total(order_item) %></em>
```

Comment:
I'm sure you're aware of this, but as of right now there aren't really prices seeded into the database. I'm assuming that in production we will always have items with associated prices so this won't be a problem in the future

Reply:
Fair, I've mocked up some prices locally, but it makes sense that they might not always be there, I can try and add some handling for scenarios without prices to this logic.

Reply:
Fixed in commit https://github.com/mojotech/lunchbot/pull/31/commits/bf49b1d4382a69f23fc19ba745986469c68846ad